mlflow/utils/autologging_utils/__init__.py

```diff
@@ -343,6 +344,10 @@ def autolog(*args, **kwargs):
             config_to_store.update(kwargs)
             AUTOLOGGING_INTEGRATIONS[name] = config_to_store
 
+            if get_autologging_config(name, "disable", True):
+                revert_patches(name)
+                return
```

Comment:
I think this logic works well for all autologging integrations except for the `mlflow.autolog()` fluent API, which makes use of the `@autologging_integration` decorator.

If we call `mlflow.autolog(disable=True)`, we should still call all of the active framework-specific autologging integrations so that their patches are reverted.

I think it's sufficient to add a special case here for `name == 'mlflow'` with an explanatory inline comment. Can we also make sure to add test coverage for this in `test_fluent_autolog.py`?

Reply:
Done, implemented logic in `revert_patches`